src/number-formatting/format.js

```diff
@@ -104,11 +106,14 @@ function formatNumber(inputNumber, decimals, options) {
     let absoluteNumber = Math.abs(inputNumber);
     absoluteNumber = Math.round(absoluteNumber * factor) / factor;
 
-    let formattedNumber = expandNumber(Math.abs(absoluteNumber),
-                                decimals,
-                                options.groupSizes,
-                                options.groupSeparator,
-                                options.decimalSeparator);
+    const opts = {
+        groupSizes: options.groupSizes,
+        sep: options.groupSeparator,

```

Comment:
We are breaking expand number contract anyway - why use different option names than the ones passed in?

I prefer the explicit groupSeparator to sep and decimalSeparator to decimalChar